src/lang/std/engineConnection.ts

```diff
@@ -307,13 +307,10 @@ class EngineConnection extends EventTarget {
   }
 
   set state(next: EngineConnectionState) {
-    console.log(`${JSON.stringify(this.state)} → ${JSON.stringify(next)}`)
```

Comment:
Please leave this one, or if you can include these messages in the coredump, that'd be ideal

Reply:
![IMG_1963](https://github.com/KittyCAD/modeling-app/assets/4322/d8842895-1e24-4ef0-ab7f-fa56389ea51e)

I added this back. I'll make sure this is properly captured in coredump.